packages/renderer/src/lib/ui/ProviderInfo.svelte

```diff
@@ -0,0 +1,39 @@
+<script lang="ts">
+import Tooltip from './Tooltip.svelte';
+
+// Name of the provider (e.g. podman, docker, kubernetes)
+export let provider = '';
+
+// Only used for Kubernetes-like distros
+export let context = '';
+
+// Each provider has a colour associated to it within tailwind, this is a map of those colours.
+// bg-purple-600 = podman
+// bg-sky-300 = docker
+// bg-sky-600 = kubernetes
+// bg-gray-900 = unknown
+function getProviderColour(providerName: string): string {
+  switch (providerName) {
+    case 'Podman':
+      return 'bg-purple-600';
+    case 'Docker':
+      return 'bg-sky-400';
+    case 'Kubernetes':
+      return 'bg-sky-600';
+    default:
+      return 'bg-gray-900';
+  }
+}
```

Comment:
We can't hardcode values based on the name of the engine. It's unknown from the core part

you could use the engine type docker vs podman but not the name

Reply:
Sure, I'll make that change. For container side, it will have to be engineType and use that instead of engineName.

But for pods side, we'll have to use kind: https://github.com/cdrage/podman-desktop/blob/b59dfc2a26156b053ace35475e351a110274a015/packages/renderer/src/lib/pod/PodInfoUI.ts#L38-L39

Reply:
yes relying on kind for k8s and engineType for container side will be more generic 🎉